src/main/java/net/mcreator/ui/modgui/RecipeGUI.java

```diff
@@ -258,8 +271,14 @@ public RecipeGUI(MCreator mcreator, ModElement modElement, boolean editingMode)
 				return new AggregatedValidationResult.FAIL(
 						L10N.t("elementgui.recipe.error_brewing_no_input_ingredient_and_result"));
 			}
+		} else if ("Anvil".equals(recipeType.getSelectedItem())) {
+			if (!arm.cb1.containsItem() || !arm.cb2.containsItem() || !arm.cb3.containsItem()) {
+				return new AggregatedValidationResult.FAIL(
+						L10N.t("elementgui.recipe.error_anvil_no_ingredient_addition_and_result"));
```

Comment:
Maybe `elementgui.recipe.error_anvil_missing_item` can make the key shorter and still be understandable

Reply:
I copied the translation key from the smithing recipe error

Reply:
It's still pretty long as a key, so I don't know if we should continue like the smithing key.